mithril-client/src/aggregator_client.rs

```diff
@@ -238,6 +238,7 @@ impl AggregatorHTTPClient {
         };
 
         let mut http_headers = HeaderMap::new();
+        http_headers.insert(ACCEPT_ENCODING, HeaderValue::from_static("gzip, deflate"));
```

Comment:
This will cover more encodings:
```suggestion
        http_headers.insert(ACCEPT_ENCODING, HeaderValue::from_static("gzip, deflate, br, zstd"));
```

Reply:
It is included and handled differently in the latest commits.